src/feature/hs/hs_descriptor.h

```diff
@@ -69,6 +69,17 @@ typedef enum {
   HS_DESC_AUTH_ED25519 = 1
 } hs_desc_auth_type_t;
 
+/* Error code when decoding a descriptor. */
```

Comment:
Perhaps let's document the individual error codes here?

Reply:
Done.